docs/modules/ROOT/pages/advanced-how-tos/building-olm.adoc

```diff
@@ -0,0 +1,27 @@
+= Building an Operator Lifecycle Manager (OLM) in {ProductName}
+
+If you are using an link:https://olm.operatorframework.io/docs/[Operator Lifecycle Manager (OLM)] for your application, you can build the OLM in {ProductName}. The first procedure in this document explains how to use {ProductName} to build an OLM in the most basic sense--building its operator and bundle. The second procedure is optional but recommended. It explains how you can automatically update the image references to the operator and operands that should be present in your bundle.
+
+
+== Builing the operator and the bundle
+
+[NOTE] 
+====
+This procedure assumes that the source code for your OLM operator and bundle, including the Dockerfile, are in the same git repository, per OLM convention.
+====
+
+.Procedure
+
+. In the {ProductName} UI,  xref:../how-tos/creating.adoc[create a new application] for your OLM in {ProductName}.
```

Comment:
Same as before. OLM is the lifecycle manager, not what you build. I would change it for "OLM operator" as it is below, or simply operator